server/src/infra/entities/user.entity.ts

```diff
@@ -1,3 +1,4 @@
+import { UserAvatarColor } from '@app/domain/user/response-dto/user-response.dto';
```

Comment:
Importing from app/domain into user entities can cause circular imports. Can you move the enum to this file instead?